.gitignore

```diff
@@ -38,3 +38,6 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+# Ignore Emacs backups files
+*~
```

Comment:
você pode usar um ignore global para necessidades particulares do seu editor. 

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer